index.html

```diff
@@ -2564,11 +2564,11 @@ <h2>Definition of Roles</h2>
 			</table>
 		</div>
 		<div class="role" id="definition">
-			<rdef>definition</rdef>
-			<div class="role-description">
-				<p>A definition of a term or concept. See related <rref>term</rref>.</p>
-				<p>Authors SHOULD identify the <a>element</a> being defined by giving that element a role of <rref>term</rref> and referencing it with the <pref>aria-labelledby</pref> <a>attribute</a> or by making the element with role <rref>term</rref> a descendant of the element with role <code>definition</code>.</p>
-			</div>
+      <rdef>definition</rdef>
+      <div class="role-description">
+        <p>A definition of a term or concept. See related <rref>term</rref>.</p>
+        <p>Authors SHOULD identify the <a>element</a> being defined and assign that element a role of <rref>term</rref>. In addition, authors SHOULD set <pref>aria-details</pref> on the <rref>term</rref>, pointing to the related <code>definition</code>.</p>
```

Comment:
This reads like something that should be in the `term` definition not the `definition` definition. Can a `definition` exist without a matching term? I'm unsure from this definition and I think we should make this clear one way or the other

Reply:
I don't know? I'd say a definition requires a term, although I believe a term could exist without a definition. 

Also, I thought of mentioning the aria-details here just to make sure people saw that and got it right. I was afraid they'd see it needs a term but not go look at that, haha.

Want me to remove the part about aria-details?

Reply:
Ok:
- I removed the sentence about aria-details and only mention it in term.
- Now using MUST to say that a definition needs a term
- Now using MUST in the description of term to say if it has a matching definition, it MUST use aria-details to point to it.

How's that?

Reply:
Now the spec for term sounds confused. The first sentence says it has "a corresponding definition". The 2nd sentence says "a definition has been provided by the author or is expected to be provided by the user". Then the 3rd sentence says "Where a matching definition exists, ...".

I'm not sure whether term can/should be standalone? Technically, html dfn can - kinda - be standalone, but it's usually within a paragraph or maybe a dl that contains its definition, and I'm not sure what it would mean semantically to have an undefined term?

Reply:
I don't think it's more confusing than what we have, which already says there is a definition OR one is expected from the user. But, I do think it's confusing. I've pushed a change that may help clear it up. LMK what you think.